### PR TITLE
Update Android stores

### PR DESCRIPTION
**Pull request by jovirkku**
The APK Pure and Aptoide do not belong to the recommended Android app stores of Jolla anymore. The latter was even in the Jolla Store. It was removed a short time ago. I have done my best to remove all mentions of these two stores in the sailfishos.org documents.

The Marketplaces of the Jolla Store has now the F-Droid and Aurora Store. They are mentioned in the docs as the recommended Android app stores. There are no dedicated documents for installing or using them.

### CODE REVIEW

**Comment by jovirkku** [comment]
In commit https://github.com/sailfishos/docs.sailfishos.org/commit/8b700438f7cf14129446d000eeadc26f4cfd218c I clarified the language in the main document of AppSupport. In addition, I fixed the text about "untrusted software".

File: Support/Help_Articles/Android_App_Support/README.md [hunk 1]
```diff
@@ -14,28 +14,27 @@ In this article, you will learn how to make the best of the **Android AppSupport
...
+Before installing any Android apps, **Android AppSupport** (simulation of the Android system on Sailfish OS) must be installed. It is available in the Jolla Store. This requires the Sailfish OS licence.
 
 Android apps are installed in the internal storage of a phone. Installing apps on SD cards is not supported.
 
-After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` shows all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps.
+After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` has all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps.
 
 ## Installing applications from application stores
-The SailfishOS operating system of Jolla supports Android apps that are featured in the Jolla Store and in Jolla-certified app partner stores.
+The SailfishOS operating system of Jolla supports Android apps that are featured in the Jolla Store and Jolla-certified app partner stores.
```
Comment (direc85) [inline Support/Help_Articles/Android_App_Support/README.md:26]:
```suggestion
Sailfish OS supports Android apps featured in Jolla Store and the Jolla-certified app partner stores.
```

Slightly off-topic, but `SailfishOS` is used here and on line 31, whereas `Sailfish OS` is used elsewhere.

File: Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md [hunk 2]
```diff
@@ -7,43 +7,43 @@ layout: default
...
+* SIM cards ("Settings > SIM cards": select the active SIM card(s) and the SIM card to be used for outgoing calls and messages)
+* Mobile network ("Settings > Mobile network > Use SIM card": select the card to be used for mobile data)
 * Language
 * Text input 
 
 # Sign in to your accounts that you want to use on this Sailfish device
 
-* Google, adding the Google account to your device is described in this [separate document about Google account](/Support/Help_Articles/Accounts_Setup/Setup_Google_Account)
-* Nextcloud (Settings > Accounts > Add account > Nextcloud)
+* Adding the Google account to your device is described in [this document](/Support/Help_Articles/Accounts_Setup/Setup_Google_Account)
```
Comment (direc85) [inline Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md:39]:
```suggestion
* Adding a Google account to your device is described in [this document](/Support/Help_Articles/Accounts_Setup/Setup_Google_Account)
```

File: Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md [hunk 3]
```diff
@@ -7,43 +7,43 @@ layout: default
...
+* Nextcloud ("Settings > Accounts > Add account > Nextcloud")
 
 # Install Android app store(s)
 
-The Sailfish OS operating system of Jolla supports Android apps that are featured in the Jolla Store and in Jolla-certified app partner stores. More information can be found from [here](/Support/Help_Articles/Android_App_Support/#installing-applications-from-application-stores). Note that Settings > Untrusted software needs to be enabled in order to install these.
-* APKPure, more information and how to install APKPure can be found from [this separate document about APKPure](/Support/Help_Articles/Android_App_Support/APKPure/)
-* [Aurora Store](https://auroraoss.com)
+The Sailfish OS operating system of Jolla supports Android apps that are featured in the Jolla Store and in Jolla-certified app partner stores. More information can be found in [this document](/Support/Help_Articles/Android_App_Support/#installing-applications-from-application-stores). Note that "Settings > Untrusted software" needs to be enabled before installing the stores or apps.
```
Comment (direc85) [inline Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md:44]:
```suggestion
Sailfish OS supports Android apps featured in Jolla Store and the Jolla-certified app partner stores. More information can be found in [this document](/Support/Help_Articles/Android_App_Support/#installing-applications-from-application-stores). Note that "Settings > Untrusted software" needs to be enabled before installing the stores or apps.
```

File: Support/Help_Articles/Android_App_Support/README.md [hunk 4]
```diff
@@ -14,28 +14,27 @@ In this article, you will learn how to make the best of the **Android AppSupport
...
-
-Another preparatory step is to enable "Settings > System > Untrusted software > Allow untrusted software". Installing Android apps (including application stores) from previously unused sources is considered potentially risky. Therefore, you must deliberately allow it.
+Before installing any Android apps, **Android AppSupport** (simulation of the Android system on Sailfish OS) must be installed. It is available in the Jolla Store. This requires the Sailfish OS licence.
 
 Android apps are installed in the internal storage of a phone. Installing apps on SD cards is not supported.
 
-After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` shows all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps.
+After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` has all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps.
```
Comment (direc85) [inline Support/Help_Articles/Android_App_Support/README.md:23]:
```suggestion
After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` has all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps. Files saved using Android apps, for example downloaded files in a web browser app or photos taken in a camera app, are stored in their respective folders in ```/home/defaultuser/android_storage```.
```

**Review by direc85 — APPROVED** [review]
Looks good overall. I added a few corrections and some suggestions, nothing blocking in those however.

**Comment by jovirkku** [comment]
I believe I have fixed the text according to the comments of direc85. Thank you for improving the language.

File: Support/Help_Articles/Android_App_Support/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Android_App_Support/README.md]: Done.

File: Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md (hunk 2 above)
Comment (jovirkku) [reply Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md]: Done.

File: Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md (hunk 3 above)
Comment (jovirkku) [reply Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/README.md]: Done.

File: Support/Help_Articles/Android_App_Support/README.md (hunk 4 above)
Comment (jovirkku) [reply Support/Help_Articles/Android_App_Support/README.md]: Done.

**Pull request merged**